shell/toSwaggerHub.sh

```diff
@@ -5,11 +5,9 @@ echo "--> toSwaggerHub.sh"
 publishToSwagger() {
     apiKey=$1
     apiFolder=$2
-    apiVersion=$3
-    oasVersion=$4
-    isPrivate=$5
```

Comment:
@cherrycl after talking with @lenny-intel I don't think removing `isPrivate` is the correct behavior.